installation/installing-images/linux.md

```diff
@@ -1,5 +1,7 @@
 # Installing operating system images on Linux
 
+[Etcher](README.md) is typically the easiest option for most users to write images to SD cards, and you should start there if you're not sure. If you want more advanced options on Linux though you can use the standard command line tools below.
```

Comment:
And I'm not sure if "If you want more advanced options..." is the right language to use? It might imply that Etcher lacks particular options, which (for the purposes here) it doesn't.
Perhaps something like "If you want to use advanced methods using low-level tools..." or something?